PR-App/PR-App/Custom Views/Labels/CustomLabel.swift

```diff
@@ -19,16 +19,15 @@ class CustomCellLabel: UILabel {
         fatalError("init(coder:) has not been implemented")
     }
     
-    init(text: String) {
-        super.init(frame: .zero)
-        configure()
+    convenience init(text: String, alignment: NSTextAlignment, size: CGFloat, weight: UIFont.Weight, color: UIColor) {
+        self.init(frame: .zero)
+        self.textAlignment = alignment
         self.text = text
+        self.font = UIFont.systemFont(ofSize: size, weight: weight)
+        self.textColor = color
     }
     
     private func configure() {
```

Comment:
Not used, I think it could be deleted.

Reply:
self.font and self.textColor is used, please check convenience initializer again.